redhat-cop/containers-quickstarts
# Update rabbitmq configuration

#### What is this PR About?
Update to latest version of RabbitMQ (3.8.3) as default build
Add RabbitMQ HA policy to sync between pods
Add Pod AntiAffinity to ensure that all pods are on different nodes
Set minimum resource limits to satisfy additional HA load
Updated Documentation in configmap entry


#### How do we test this?
Provide commands/steps to test this PR.
1. Deploy as you normally do.
2. Attach connect to an application that uses the default queue "/"
3. Run this command in terminal to ensure the queue is being synced between three pods
    rabbitmqctl list_queues name slave_pids synchronised_slave_pids
 
### Other Notes:
I deployed in my Openshift instance and confirmed working.

cc: @redhat-cop/day-in-the-life
